browser/lib/transport/xhrtransport.js

```diff
@@ -9,7 +9,7 @@ var XHRTransport = (function() {
 	XHRTransport.isAvailable = XHRRequest.isAvailable;
 
 	XHRTransport.checkConnectivity = function(callback) {
-		Http.Request('http://internet-up.ably.io.s3-website-us-east-1.amazonaws.com/is-the-internet-up.txt', null, null, null, function(err, responseText) {
+		Http.Request(Defaults.internetUpUrlWithoutExtension + '.txt', null, null, null, function(err, responseText) {
```

Comment:
Ah, I see there are 2 different `Defaults.js` files anyway, so it's a very easy fix